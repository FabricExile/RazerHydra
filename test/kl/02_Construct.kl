require RazerHydra;

operator entry() {
  RazerHydraController controller();
  controller = null;
  report('completed!'); 
}