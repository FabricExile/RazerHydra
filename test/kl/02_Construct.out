[FABRIC:MT] RazerHydra: Controller initialized.
[FABRIC:MT] RazerHydra: Controller shutdown.
[FABRIC:MT] completed!
